src/Scripty.CustomTool/BaseCodeGenerator.cs

```diff
@@ -1,4 +1,6 @@
-﻿
+﻿//Dave Glick - is this just template copyright stuff that was brought in or 
+// is this something else?
+
```

Comment:
Good call - planning on continuing work on updating everything tomorrow so I'll take a look and see if they can be stripped.